root/etc/s6-overlay/s6-rc.d/init-outdated-config/run

```diff
@@ -11,9 +11,9 @@ if [[ -f /config/nginx/ldap.conf ]]; then
         Ensure your configs are updated and remove /config/nginx/ldap.conf
         If you do not use this config, simply remove it."
 fi
-if grep -qrle ' /etc/letsencrypt' /config/nginx; then
+if grep -qr '/etc/letsencrypt' --include="*.conf" /config/nginx; then
     echo "    The following nginx confs are using certificates from the obsolete location
     /etc/letsencrypt and should be updated to point to /config/etc/letsencrypt
         "
-    echo -n "    " && grep -rle ' /etc/letsencrypt' /config/nginx
+    echo -n "    " && grep -r '/etc/letsencrypt' --include="*.conf" /config/nginx | grep >
```

Comment:
Removing `e` achieves nothing, it's the pattern signifier. Removing `l` means it prints output for *every reference* in *every* file, instead of just printing each matching file once.

Piping the output to `grep >` is just going to error because you're trying to redirect output to a newline.

I'm unclear both what the original issue is, and how these changes are supposed to fix it.

Reply:
More importantly, changing the pattern being matched from `' /etc/letsencrypt'` to `'/etc/letsencrypt'` (removing the leading space inside the quotes) would actually create a new issue because this new `grep` command will match configs that contain `/config/etc/letsencrypt`

As a side note, I don't recall any nginx conf we shipped that used `'/etc/letsencrypt'` or `'/config/etc/letsencrypt'`. There are however certbot configs used for renewal that should match the glob `/config/etc/letsencrypt/renewal/*.conf` that look to be handled in https://github.com/linuxserver/docker-swag/blob/master/root/migrations/02-swag-old-certbot-paths

Reply:
*We* didn't ship any, but a bunch of people had created their own confs and included the cert paths, which is why the check/log warning was added.